26-register_registration/creator_navigates_to_description_tab.feature

```diff
@@ -48,3 +49,19 @@ Feature: Creator navigates to Description tab
     Given Creator has added a Project
     When they click the Remove Project icon
     Then they see the Project is removed from the list of selected Projects
+
+  @2446
+  Scenario: Creator adds an Allowed Vocabulary
+    Given Creator begins registering a Registration in the Wizard
+    And they navigate to the Description tab
+    And their institution has a Vocabulary set as "Allowed"
+    When they click "Add controlled Vocabulary"
```

Comment:
når man klikker på `controlled vocabulary` så kan de legge til `allowed vocabulary`? dette er første gang `controlled` er brukt?

Reply:
Det var basert på skisser. men kan bare kalle det "Add Vocaulary", vil ikke funke for string matching, men er likevel mer tydelig når man leser scenario